conf/broker.conf

```diff
@@ -240,6 +240,9 @@ authenticateOriginalAuthData=false
 # Deprecated - Use webServicePortTls and brokerServicePortTls instead
 tlsEnabled=false
 
+# Tls cert refresh duration in seconds (set 0 to check on every new connection) 
+tlsCertRefreshCheckDurationSec=300
```

Comment:
Don't we already have a refresh time from #3568?

Reply:
yes, we missed to define into conf files.